charts/managed-identity-wallets/Chart.yaml

```diff
@@ -15,8 +15,8 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.8
-appVersion: 3.1.2
+version: 0.6.9
```

Comment:
When bumping the minor version in appVersion consider to bump minor version for Chart version instead of the patch version.